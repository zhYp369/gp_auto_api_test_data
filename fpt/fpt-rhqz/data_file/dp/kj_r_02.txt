bodydata>REQUEST_COMMON_FPKJ.YFP_DM:kj_b_01>response.interface.data.content.DATA.FP_DM
bodydata>REQUEST_COMMON_FPKJ.YFP_HM:kj_b_01>response.interface.data.content.DATA.FP_HM

