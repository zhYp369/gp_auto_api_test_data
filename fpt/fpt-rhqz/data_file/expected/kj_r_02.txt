"returnCode":"0000","returnMessage":"成功"
"returnCode":"0000","returnMessage":"成功"